site/smart-accounts/signers/particle-network.md

```diff
@@ -0,0 +1,81 @@
+  - - meta
+    - property: og:description
+      content: Guide to use Particle Network as a signer
+---
+
+# Particle Network
+
+[**Particle Network**](https://particle.network/) is the Intent-Centric, Modular Access Layer of Web3. With Particle's Wallet-as-a-Service, developers can curate unparalleled user experience through modular and customizable embedded wallet components. By utilizing MPC-TSS for key management, Particle can streamline onboarding via familiar Web2 accounts—such as Google accounts, email addresses, phone numbers, etc.
+
+Leveraging both Particle and Account Kit enables a streamlined onboarding flow, with social logins and signer key management being handled by Particle while Account Kit takes this experience to the next level with account abstraction - facilitating powerful user experience.
+
+## Integration
+
+### Sign up for a Particle Account
+
+To configure Particle, you'll need to start by quickly signing up for a Particle account, creating a project, and then creating an application. You can learn more about this process within our [quickstart guide](https://docs.particle.network/getting-started/dashboard/manage-projects). Additionally, you can sign up through the [Particle dashboard](https://dashboard.particle.network/#/login).
+
+### Install the SDK
+
+::: code-group
+
+```bash [npm]
+npm i -s @particle-network/auth
+npm i -s @particle-network/provider
+```
+
+```bash [yarn]
+yarn add @particle-network/auth
+yarn add @particle-network/provider
+```
+
+:::
+
+### Create a SmartAccountSigner
+
+With `@particle-network/auth` and `@particle-network/provider` installed, you can move on to creating a `SmartAccountSigner`. To do this, you'll need to ensure you have your `projectId`, `clientKey`, and `appId` from the Particle dashboard to use in the configuration of `ParticleNetwork`.
+
+From here, setting up a `SmartAccountSigner` involves the initialization of `ParticleProvider` to be used in a viem wallet client, which then gets passed into our `SmartAccountSigner`.
+
+<<< @/snippets/particle.ts
+
+### Use it with LightAccount
+
+Let's see it in action with `aa-alchemy` and `LightSmartContractAccount` from `aa-accounts`:
+::: code-group
+
+```ts [example.ts]
+import { AlchemyProvider } from "@alchemy/aa-alchemy";
+import { LightSmartContractAccount } from "@alchemy/aa-accounts";
+import { polygonMumbai } from "viem/chains";
+import { particleSigner } from "./particleSigner";
+
+const chain = polygonMumbai;
+const provider = new AlchemyProvider({
+  apiKey: process.env.ALCHEMY_API_KEY,
+  chain,
+  entryPointAddress: ENTRY_POINT_CONTRACT_ADDRESS, // 0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789
+}).connect(
+  (rpcClient) =>
+    new LightSmartContractAccount({
+      entryPointAddress: ENTRY_POINT_CONTRACT_ADDRESS,
+      chain: rpcClient.chain,
+      owner: particleSigner,
+      factoryAddress: FACTORY_CONTRACT_ADDRESS, // For LightAccount, 0x000000893A26168158fbeaDD9335Be5bC96592E2
```

Comment:
let's use the `get*` method we have for light account

Reply:
oh right, just copy pasted from #152, updated